src/claim/Claim.ts

```diff
@@ -38,6 +38,24 @@ export default class Claim implements IClaim {
     return new Claim(claimInput)
   }
 
+  public static fromNestedCTypeClaim(
+    cTypeInput: ICType,
+    nestedCType: Array<ICType['schema']>,
+    claimContents: object,
+    claimOwner: IPublicIdentity['address']
+  ): Claim {
+    if (
+      !CTypeUtils.compileSchema(cTypeInput.schema, nestedCType, claimContents)
+    ) {
+      throw Error('Claim data doesnt match and not valid')
```

Comment:
Do you think the wording is incorrect and doesn't correspond to what is happening if an Error is found?

Reply:
I have updated, hope this represents it better.